benchmark/url/new-url-properties.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+const common = require('../common.js');
+const url = require('url');
+const URL = url.URL;
+const v8 = require('v8');
+const assert = require('assert');
+
+var inputs = {
+  long: 'http://nodejs.org:89/docs/latest/api/url.html#test?' +
+        'payload1=true&payload2=false&test=1&benchmark=3&' +
+        'foo=38.38.011.293&bar=1234834910480&test=19299&3992&' +
+        'key=f5c65e1e98fe07e648249ad41e1cfdb0',
+  short: 'https://nodejs.org/en/blog/',
+  idn: 'http://你好你好',
+  auth: 'https://user:pass@example.com/path?search=1',
+  special: 'file:///foo/bar/test/node.js'
+};
+
+const bench = common.createBenchmark(main, {
+  type: Object.keys(inputs),
+  prop: ['stringifier', 'protocol', 'auth', 'host', 'hostname', 'port',
+         'pathname', 'search', 'hash'],
+  method: ['old', 'new'],
```

Comment:
I know this is a vestige of `new-url-parse`, but I think Node.js/legacy vs WHATWG is clearer.

Also the prop name `method` doesn't really convey the difference here, I think.